Google.PowerShell.IntegrationTests/Storage/Storage.GcsObject.Tests.ps1

```diff
@@ -207,6 +212,66 @@ Describe "New-GcsObject" {
         $obj.Metadata["Content-Type"] | Should Be "image/jpeg"
         Remove-GcsObject $obj
     }
+
+    It "should upload an empty folder" {
+        $TestFolder = Join-Path $TestDrive "TestFolder"
```

Comment:
The naming convention we have been using for PowerShell is to have variable names camelCased (and not PascalCased). Is that the standard for PowerShell? If so, let's talk about this in-person as we should be consistent for now and do the major refactoring in a separate PR.